### ISSUE TITLE
BUG: Metasploit URL Incorrect

### ISSUE THREAD

**Issue by snow-lab** [fireeye/commando-vm#89]
**Describe the bug and expected behavior**
Metasploit framework URL is out of date by a couple weeks. The installation script is expecting 2917-07-14, but the latest one would be 2019

**To Reproduce**
1. choco install metasploit.flare 
2. script throws a 404 error 

**Screenshots**
![image](https://user-images.githubusercontent.com/53309860/62505148-3d2d0b00-b7b0-11e9-808a-37981ee57a2a.png)

**Version**
 - OS: Win10

**Additional context**
I think Rapid7's windows page (http://windows.metasploit.com/) should be polled for changes and automatically updated accordingly.

**Comment by Milenko22** [comment]
Hi there, I'm having the same problem as snow-lab. I've checked the URL for the metasploit msi and it doesn't exist. I have a handful of tools that aren't installing. I think for the same reason. Is there a way to manually install the latest for the tools? When I run the installer it is obviously pulling the package versions first which then doesn't correspond in some cases. I tried editing the .nuspec file but that gets overwritten during the install. Any help?

Thanks.
